chain/epoch_manager/src/lib.rs

```diff
@@ -644,6 +653,27 @@ impl EpochManager {
     pub fn get_epoch_inflation(&mut self, epoch_id: &EpochId) -> Result<Balance, EpochError> {
         Ok(self.get_epoch_info(epoch_id)?.inflation)
     }
+
+    /// Compare two epoch ids based on their start height. This works because finality gadget
+    /// guarantees that we cannot two different epochs on two forks
+    pub fn compare_epoch_id(
+        &mut self,
+        epoch_id: &EpochId,
+        other_epoch_id: &EpochId,
+    ) -> Result<Ordering, EpochError> {
+        if epoch_id.0 == other_epoch_id.0 {
```

Comment:
What if both epochs are equal but unknown? This will return `Ok(Ordering::Equal)`, instead of `Err`. This is not a problem, but should be written in comments maybe.

Reply:
Do you think we should return error instead?

Reply:
Yes.